README.md

```diff
@@ -1,23 +1,94 @@
-# Onigumo #
+# Onigumo
 
-## About ##
+## O projektu
 
-This is an attempt to build just another web-crawler, also called _Spider_. Its purpose is to get data from a website in a form of a list of objects. This data can be then used to download linked files and place them in a database or a folder structure.
+Onigumo je jeden z dalších web-crawlerů, také známých pod pojmem _Spider_.
+Onigumo obecně řídí toky dat, určuje směry toků a v jaké chvíli se
+aktivuje příslušná operace.
+Konrétně slouží k procházení webových aplikací či statických webových stránek.
+Obsah a případně další příslušná metadata uloží do strukturované podoby,
+která je vhodná pro další strojové zpracování. K dosažení tohoto cíle je
+Onigumo rozděleno do tří vzájemně propojených logických celků
+(Operator, Downloader, Parser), které tvoří procesní workflow.
```

Comment:
I don’t understand the term “procesní workflow”.

Reply:
pracovní tok procesů, popravdě je to termín který jsem někde pochytil... Myslím, že jsem jej pochytil při prezentaci IS pro naši firmu. Ale ve skutečnosti nevím co má konkrétně znamenat. Klidně jej zrušme